app/experimenter/nimbus-ui/src/components/PageNew/index.tsx

```diff
@@ -7,6 +7,8 @@ import { RouteComponentProps } from "@reach/router";
 import AppLayout from "../AppLayout";
 import LinkExternal from "../LinkExternal";
 import FormOverview from "../FormOverview";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
```

Comment:
Nit: all bootstrap components can come from one import

```tsx
import { Row, Col } from "react-bootstrap";
```

Reply:
[You should import individual components like: react-bootstrap/Button rather than the entire library.](https://react-bootstrap.github.io/getting-started/introduction/#importing-components)

Reply:
TIL!